tests/test_username_api.py

```diff
@@ -40,405 +45,87 @@ def get_expected_response(website, user, status):
               else None
   }
 
+def load_availability_test_cases(status):
+  res = []
+  for website in data:
+    if status == 'available_username':
+      usernames = data[website]['available_usernames']
+    if status == 'taken_username':
+      usernames = data[website]['taken_usernames']
+    res.extend((website, user) for user in usernames)
+
+  return res
+
+def load_format_checking_cases():
+  res = []
+  for website in data:
+    if website == 'facebook':
```

Comment:
data belongs in the yaml, not here in code.

Reply:
umm... what do you mean?

Reply:
put these usernames in https://github.com/manu-chroma/username-availability-checker/blob/master/tests/test_data.yml

Reply:
got it

Reply:
why don't we add new key, `invalid_usernames`? then we can remove this if statement.